docs/standard/base-types/regular-expression-options.md

```diff
@@ -11,12 +11,11 @@ helpviewer_keywords:
   - ".NET regular expressions, options"
   - "inline option constructs"
   - "options parameter"
-ms.assetid: c82dc689-7e82-4767-a18d-cd24ce5f05e9
 ---
 
-# Regular Expression Options
+# Regular expression options
```

Comment:
The below table is also missing the new RegexOptions.NonBacktracking being added in .NET 7.  Presumably we should add that as part of a separate PR.